oam/install_scripts/mcs-storagemanager.service.in

```diff
@@ -1,16 +1,7 @@
 [Unit]
 Description=storagemanager
-PartOf=mcs-workernode.service
-Before=mcs-workernode.service
-ConditionPathExists=/etc/columnstore/storagemanager.cnf
-# FailureAction="exit"
-# FailureActionExitStatus=0
 
 [Service]
 Type=simple
-ExecStartPre=@ENGINE_BINDIR@/mcs-start-storagemanager.py
 ExecStart=/usr/bin/env bash -c "LD_PRELOAD=$(ldconfig -p | grep -m1 libjemalloc | awk '{print $1}') exec @ENGINE_BINDIR@/StorageManager"
-ExecStop=/usr/bin/env bash -c "kill -15 $MAINPID"
-
-[Install]
-WantedBy=mcs-workernode.service
+SuccessExitStatus=255
```

Comment:
sending a term signal to storagemanager causes an exit status of 255. let's handle this as success for now since this isn't an actual problem with the unit